src/modules/app/components/App.js

```diff
@@ -32,19 +50,27 @@ class App extends React.Component<{}, { waitingForStore: boolean }> {
       return null
     }
 
+    Sentry.captureBreadcrumb({
+      message: 'App rendered',
+      category: 'component',
+      data: {}
+    })
+
     return (
-        <Provider store={this.store}>
-          <I18nProvider>
-            <CustomThemeProvider>
+      <Provider store={this.store}>
+        <I18nProvider>
+          <CustomThemeProvider>
+            <Dialog>
```

Comment:
What kind of Dialog is this? (Adjust naming)

Reply:
Renamed to LaunchInquiry